static/app/router/routes.tsx

```diff
@@ -1213,7 +1213,7 @@ function buildRoutes(): RouteObject[] {
     },
     {
       path: 'seer/',
-      name: t('Seer Automation'),
+      name: t('Seer'),
```

Comment:
This name affects the breadcrumbs along the top of settings:

<img width="839" height="184" alt="Image" src="https://github.com/user-attachments/assets/e01227f0-3a01-4a04-9279-0dc562ba71ce" />

I can't seem to override things, so we'll set "Seer" at the root, and sub-pages like Projects and Repos don't need to set anything at all. All pages will have the same crumbs with this PR